.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,78 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Needed for micromamba pickup
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  N_LATEST_UPLOADS: 5
+  ANACONDA_USER: "scientific-python-nightly-wheels"
+
+jobs:
+  remove:
+
+    runs-on: ubuntu-latest
+    environment:
+      name: remove-old-wheels
+
+    steps:
+      - name: Install micromamba and anaconda-client
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: remove-wheels
+          create-args: >-
+            python=3.11
+            anaconda-client=1.11.2
+
+      - name: Show environment
+        run: env
+
+      - name: Show CLI API info
+        run: |
+          anaconda show --help
+          echo ""
+          anaconda remove --help
+
+      - name: Query package index for packages
+        run: |
+          anaconda show "${ANACONDA_USER}" &> >(grep "${ANACONDA_USER}/") | \
+              awk '{print $1}' | \
+              sed 's|.*/||g' > package-names.txt
```

Comment:
Curious what the sed does here; would expect awk to print only the relevant part of what you need.

Reply:
So indeed `awk` will given you the `USER/PACKAGE` information that is what is needed later

```console
$ anaconda show scientific-python-nightly-wheels &> >(grep "scientific-python-nightly-wheels/") | \
    awk '{print $1}'
scientific-python-nightly-wheels/ipython
scientific-python-nightly-wheels/matplotlib
scientific-python-nightly-wheels/networkx
scientific-python-nightly-wheels/numpy
scientific-python-nightly-wheels/scikit-image
scientific-python-nightly-wheels/scikit-learn
scientific-python-nightly-wheels/scipy
scientific-python-nightly-wheels/xarray
```

However, if you want to be able to refer to the package names by themselves without the user information then you would need to strip all of that off. This is what the use of `sed` does.

```console
$ anaconda show scientific-python-nightly-wheels &> >(grep "scientific-python-nightly-wheels/") | \
    awk '{print $1}' | \
    sed 's|.*/||g'
ipython
matplotlib
networkx
numpy
scikit-image
scikit-learn
scipy
xarray
```

There's other ways to do this of course, but as you've also noticed I reach for `sed` a lot out of habit/comfort.

Reply:
> also works, but as written (maybe there was a typo in the comment?)

I don't have anaconda, so there maybe multiple white spaces that the first awk works well on, but not the second. Either case I agree, more of us prefer awk than sed, and not because of performance 😄

Reply:
>I don't have anaconda

c.f. the self contained example I wrote and included in the PR body that uses Docker.

Reply:
sure, my point was that I don't have the anaconda-client to check on the whitespace in the command above. 

Bottom line, it should all be doable with one awk, but I suppose we bike shedded on it enough. Both Stefan and I made comments that made the shell more readable while using fewer tools and tricks, if you don't like the feedback, well, it's OK.